docs/rules/no-restricted-v-on.md

```diff
@@ -0,0 +1,109 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-restricted-v-on
+description: disallow specific argument in `v-on`
+---
+# vue/no-restricted-v-on
+
+> disallow specific argument in `v-on`
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule allows you to specify `v-on` argument names that you don't want to use in your application.
+
+## :wrench: Options
+
+This rule takes a list of strings, where each string is a argument name or pattern to be restricted:
+
+```json
+{
+  "vue/no-restricted-v-on": ["error", "foo", "bar"]
+}
+```
+
+<eslint-code-block :rules="{'vue/no-restricted-v-on': ['error', 'foo', 'bar']}">
+
+```vue
+<template>
+  <!-- ✗ BAD -->
```

Comment:
Can you please

1. add a "GOOD" example (just in this code block, above the "BAD" lines)
2. add back a pattern example (e.g. `"/^bar/"`)

Reply:
Sorted 💪🏾